PowerUp/app/src/main/java/powerup/systers/com/IntroActivity.java

```diff
@@ -0,0 +1,64 @@
+package powerup.systers.com;
+
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.graphics.Color;
+import android.os.Bundle;
+
+import com.github.paolorotolo.appintro.AppIntro;
+import com.github.paolorotolo.appintro.AppIntroFragment;
+
+public class IntroActivity extends AppIntro {
+
+    private SharedPreferences settings;
+    private boolean isfirstRun;
+
+    @Override
+    public void init(Bundle savedInstanceState) {
+
+        settings = getSharedPreferences("prefs", 0);
+        isfirstRun = settings.getBoolean("firstRun", true);
```

Comment:
I am aware that it may be used in other places, so feel free not to change it if you are unsure about where it is used and I will create a separate class with all the preferences strings later to prevent errors when these changes are performed